main.js

```diff
@@ -90,22 +90,30 @@ const startRound = () => {
   doorImage3.src = closedDoorPath;
   numClosedDoors = 3;
   currentlyPlaying = true;
-  startButton.innerHTML = 'Good luck!';
+  startButton.innerHTML = "Good luck!";
   randomChoreDoorGenerator();
 };
 
 const gameOver = (status) => {
-  if (status === 'win') {
-    startButton.innerHTML = 'You win! Play again?';
+  if (status === "win") {
+    startButton.innerHTML = "You win! Play again?";
     winsDisplay.innerHTML = ++wins;
   } else {
-    startButton.innerHTML = 'Game over! Play again?';
+    startButton.innerHTML = "Game over! Play again?";
     lossesDisplay.innerHTML = ++losses;
   }
   currentlyPlaying = false;
 };
 
 startRound();
 
+const isHTTPS = () => window.location.protocol === "https:";
 
-
+if (isHTTPS() && "serviceWorker" in navigator) {
```

Comment:
@aoliveiramagalhaes this adds support for installing the PWA, but only for HTTPS, meaning that, only for the deployed version.